plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -101,23 +101,22 @@ exports.onCreatePage = ({ page, actions }) => {
   }
 };
 
-const createRootNav = async ({ createNodeId, nodeModel }) => {
-  const nav = await nodeModel.runQuery({
-    type: 'NavYaml',
-    query: {
-      filter: {
-        rootNav: { eq: true },
-      },
-      sort: {
-        fields: ['title'],
-        order: ['ASC'],
-      },
-    },
-  });
+const createRootNav = async ({ args, createNodeId, nodeModel }) => {
+  const { slug } = args;
+
+  const rootNavYamlNode = nodeModel
+    .getAllNodes({ type: 'NavYaml' })
+    .filter((node) => node.path === '/');
```

Comment:
Since we have the `rootNav` boolean property on the nav yaml, can we use that instead of string based check?
```suggestion
    .filter((node) => node.rootNav);
```

Reply:
I did think of that, but i was afraid that if someone added `rootNav: true` to a nav file it would muss things up, but i guess they could also change the `path` in it, i shall accept this change and remove any mention of `rootNav` from the other navs, we are setting it to false automatically if it's not found anyway